deepchecks/nlp/utils/text_properties.py

```diff
@@ -870,44 +900,47 @@ def calculate_builtin_properties(
     )
     import_warnings = set()
 
-    progress_bar = create_progress_bar(
-        iterable=list(raw_text),
-        name='Text Samples Calculation',
-        unit='Text Sample'
-    )
-    for text in progress_bar:
-        progress_bar.set_postfix(
-            {'Sample': truncate_string(text, max_length=20) if text else 'EMPTY STRING'},
-            refresh=False
-        )
-        if pd.isna(text):
+    with tqdm(total=int(len(raw_text)//batch_size)) as pbar:
+        for i in range(0, len(raw_text), batch_size):
+            batch = raw_text[i:i + batch_size]
+            batch_properties = defaultdict(list)
+            pbar.update(1)
+
+            # filtering out empty sequences
+            filtered_sequences = [seq for seq in batch if pd.isna(seq) is False]
```

Comment:
what do you do with them? return none in the relevant places?

Reply:
np.nan. as it was before